.github/workflows/ci.yml

```diff
@@ -38,7 +38,9 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ["2.7", "3.7", "3.8", "3.9", "3.10", "3.11"]
+        # Python versions used for testing. Any changes made here should also be reflected in the noxfile.py file.
+        python-version:
+          ["2.7", "3.4", "3.5", "3.6", "3.7", "3.8", "3.9", "3.10", "3.11"]
```

Comment:
Version matrix can be kept the same. Python versions 3.5 and 3.6 need a specific os and hence are already included in line 47 and line 50 of this file.